docker-compose-dev.yml

```diff
@@ -10,11 +10,33 @@ networks:
   zero:
 
 services:
+
+  flowdb:
+    container_name: flowdb
+    image: flowminder/flowdb:master
+    ports:
+      - ${FLOWDB_PORT:-9000}:5432
+    environment:
+      - POSTGRES_DB=${POSTGRES_DB:-flowdb}
+      - POSTGRES_USER=${POSTGRES_USER:-flowdb}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-flowflow}
+      - FM_PASSWORD=${FM_PASSWORD:-foo}
+      - API_PASSWORD=${API_PASSWORD:-foo}
+    tty: true
+    stdin_open: true
+    restart: always
+    volumes:
+      - data_volume_flowdb:/var/lib/postgresql/data
+    networks:
+      db:
+        aliases:
+          - flowdb
```

Comment:
Is these both having the same alias going to cause chaos if you bring both up at once?

Reply:
Doesn't look like it. I can bring them all up and access them on the various ports via flowmachine without apparent.

Reply:
Actually, on second look it appears that if I `ping flowdb` from within the flowmachine container (while all three flowdb instances are spun up), it is not predictable _which_ of them is being pinged (and this can change randomly after a while).

Do we rely on the alias being available somewhere? I think I'd prefer to take these aliases out for the time being because otherwise it may cause some _really_ hard to track down bugs... Any objections? Happy to re-think if we _do_ need the alias name somewhere.